geometry_msgs/PointStamped point
std_msgs/String note # info about cnn or semantik
int32 clsID
float32 confidence